CONTRIBUTING.md

```diff
@@ -122,6 +122,10 @@ Because we will only merge a Pull Request for which all tests pass. The followin
 
 If you have missed a step, don't worry, the Continuous Integration will run a thorough test on your commits and the maintainers of the project can assist.
 
+Lastly, if your pull request is for resolving a certain problem for which there is an open issue, make sure to link the PR to that issue.
```

Comment:
This could be trimmed down, but it's a minor thing.

```suggestion
If your pull request addresses an open issue, make sure to link the PR to that issue.
```